psiminer-core/src/main/kotlin/labelextractor/CodeCompletionLabelExtractor.kt

```diff
@@ -0,0 +1,12 @@
+package labelextractor
+
+import GranularityLevel
+import com.intellij.psi.PsiElement
+import psi.language.LanguageHandler
+
+class CodeCompletionLabelExtractor : LabelExtractor() {
```

Comment:
Since this label extractor doesn't add specific logic, then there is no reason to name it like this. I prefer something like `DummyLabelExtractor` or `EmptyLabelExtractor`.